src/synology_dsm/api/photos/__init__.py

```diff
@@ -26,19 +26,26 @@ async def get_albums(
         """Get a list of all albums."""
         albums: list[SynoPhotosAlbum] = []
         raw_data = await self._dsm.get(
-            self.BROWSE_ALBUMS_API_KEY, "list", {"offset": offset, "limit": limit}
+            self.BROWSE_ALBUMS_API_KEY,
+            "list",
+            {"offset": offset, "limit": limit, "category": "normal_share_with_me"},
         )
         if not isinstance(raw_data, dict) or (data := raw_data.get("data")) is None:
             return None
 
         for album in data["list"]:
             albums.append(
-                SynoPhotosAlbum(album["id"], album["name"], album["item_count"])
+                SynoPhotosAlbum(
+                    album["id"],
+                    album["name"],
+                    album["item_count"],
+                    album["passphrase"] if album["passphrase"] != "" else None,
```

Comment:
```suggestion
                    album["passphrase"] if album["passphrase"] else None,
```

an empty string is always considered to be `False`